### PR TITLE
Consider no_color environment variable

### LINKED ISSUE #14: TVrank should respect the `NO_COLOR` environment variable
When the `NO_COLOR` environment variable is set, TVrank should refrain from displaying colors (and perhaps even displaying unicode art for tables).

See https://no-color.org/ for more information.

Additionally, a command-line parameter (e.g. `--color`) should be added to override the `NO_COLOR` environment variable.

Ideally, `--color` should take one of the following values:
* `on` (the default) means that color and unicode art is output only when `stdout` is a terminal.
* `off` means to never output color and unicode art.
* `always` means to always output color and unicode art even when `stdout` is not a terminal.

### PR DESCRIPTION
**Pull request by caglaryucekaya**
Resolves #14

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -438,13 +446,18 @@ fn run(cmd: Command, opt: Opts) -> Res<()> {
   debug!("Loaded IMDB database in {}", format_duration(Instant::now().duration_since(start_time)));
 
   let start_time = Instant::now();
+  let color: bool = env::var("NO_COLOR").is_err() || opt.color;
```
Comment (fredmorcos) [inline src/main.rs:449]: Can you move this to `main()` instead? Perhaps something like `color: env::var("NO_COLOR").is_err() || opts.color || opt.opts.color`?

File: src/search.rs [hunk 2]
```diff
@@ -223,16 +227,56 @@ fn create_table() -> Table {
...
+    if color {
+      Cell::new("IMDB ID").with_style(Attr::Bold)
+    } else {
+      Cell::new("IMDB ID")
+    },
+    if color {
+      Cell::new("IMDB Link").with_style(Attr::Bold)
+    } else {
+      Cell::new("IMDB Link")
+    },
```
Comment (fredmorcos) [inline src/search.rs:279]: Could you put this `if ... else ...` in a `macro_rules! make_bold` above that takes the `$title` and `$color` as arguments?

File: src/search.rs [hunk 3]
```diff
@@ -223,16 +227,56 @@ fn create_table() -> Table {
   table.set_format(table_format);
```
Comment (fredmorcos) [inline src/search.rs:228]: You can put the macro here.

**Review by fredmorcos — CHANGES_REQUESTED** [review]
Thank you so much, this is really awesome! 👍🏽 A few small changes and it can be merged 😃

**Comment by caglaryucekaya** [comment]
Done! Hope I didn't misunderstand where to declare the macro. If so, I'll move it.

**Review by fredmorcos — APPROVED** [review]
Awesome, thank you! 👍🏽

**Pull request merged**